android/build.gradle

```diff
@@ -1,4 +1,4 @@
-group 'com.plugin.example.piano_analytics_plugin'
+group 'com.plugin.piano_analytics_plugin'
 version '1.0-SNAPSHOT'
```

Comment:
Set the version

Reply:
The version of this plugin is set to pubspec.yaml. I just refacto for delete the exemple folder